### PR TITLE
Geodesy ml v0.4

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

**Comment by heyagosper** [comment]
have fun reviewing this sucker

File: gws-core/src/main/resources/db/geodesy-database-changelog.xml [hunk 1]
```diff
@@ -1,49 +1,55 @@
...
-    <include file="db/rename-responsible-party-column-name.sql"/>
-    <include file="db/drop-non-null-constraint.xml"/>
-    <include file="db/drop-old-responsible-party-table.xml"/>
-    <include file="db/add-table-column-comments-part1.xml"/>
-    <include file="db/add-table-column-comments-part2.xml"/>
-    <include file="db/add-table-column-comments-part3.xml"/>
-    <include file="db/add-delete-column-in-sitelog.xml"/>
-    <include file="db/add-user-registration.xml"/>
-    <include file="db/add-user-registration-remarks.xml"/>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```
Comment (oehm-smith) [inline gws-core/src/main/resources/db/geodesy-database-changelog.xml]: My thoughts when it comes to formatting is to either do up front and PR that before doing a PR for the changes.  Or do the PRS the other way around (because one doesn't always realise the formatting is different).  The trouble when you do both in the same PR is that it makes it difficult to see the actual changes.

**Comment by oehm-smith** [comment]
I can't add a comment to files not showing lines of code so adding as comment to conversation.

Do you really want to commit files like `.../ga/schemas/geodesyml/0.3_old/siteLog.xsd`?

**Comment by oehm-smith** [comment]
Apart from my comments, :+1:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:  :zap:

File: gws-core/src/main/resources/db/geodesy-database-changelog.xml (hunk 1 above)
Comment (lbodor) [inline gws-core/src/main/resources/db/geodesy-database-changelog.xml]: I agree with Brooke. Reformatting changes are best done in separate pull requests. It's not obvious below that both add-user-registration scripts have been removed.
